Model/Cron.php

```diff
@@ -300,7 +300,7 @@ public function execute()
         $dateStart = new \DateTime();
         $dateStart->modify('-5 day');
         $dateEnd = new \DateTime();
-        $dateEnd->modify('-1 minute');
```

Comment:
why is this different from https://github.com/Adyen/adyen-magento2/blob/develop/Model/Cron.php#L303 ?